package.json

```diff
@@ -3,17 +3,21 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@babel/runtime": "^7.18.6",
```

Comment:
Renovate will have to pin these dependencies - it might be easier to just pin them here so renovate doesn't get angry 😆

Reply:
Actually that dependency in particular I don't need, I must have accidentally copied it over so I'll remove it. So I'm a bit unclear, what dependencies need to be pinned and do I do it or Renovate?

Reply:
Renovate enforces the practice that we should specify specific version numbers (as Renovate will handle any upgrades) so if remove the caret from any dependencies you install it saves renovate opening a PR to pin them :) It's no big deal if you forget through as renovate will do it otherwise - just saves the extra PR

Reply:
Right gotcha. I'll have to get in the habit of specifying `yarn add -E <package_name>`